support-frontend/typescript-errors.json

```diff
@@ -442,6 +395,10 @@
             "path": "assets/hocs/withError.tsx",
             "message": "Type 'Omit<AugmentedProps<Props>, \"error\">' is not assignable to type 'IntrinsicAttributes & Props & { children?: ReactNode; }'."
         },
+        {
+            "path": "assets/hocs/withLabel.tsx",
+            "message": "Type 'Option<string>' is not assignable to type 'string | undefined'."
+        },
```

Comment:
Is this a knock-on effect of the changes to `label.tsx`?

Reply:
Yeah I think this will be resolved with a merge or rebase